src/xc_integrator/local_work_driver/host/reference_local_host_work_driver.cxx

```diff
@@ -299,7 +299,7 @@ void ReferenceLocalHostWorkDriver::eval_uvvar_mgga_rks( size_t npts, size_t nbe,
   const double* basis_eval, const double* dbasis_x_eval,
   const double *dbasis_y_eval, const double* dbasis_z_eval, const double* lbasis_eval,
   const double* X, size_t ldx, const double* mmat_x, const double* mmat_y, 
-  const double* mmat_z, size_t ldm,
+  const double* mmat_z, size_t /*ldm*/,
```

Comment:
This is actually a bug that should be addressed separately - we're not using the `ldm` but we should be to act as intended. Thanks for pointing it out. It would be a big help if you could propagate this, but if not, I can put it on my list.

Reply:
While I can guess what `ldm` is supposed to do from looking at other functions, I don't know for sure. So I don't feel comfortable making the change on my own.

Reply:
No problem, that's fine, I'll take it as an action item - I know this is annoying, but could you revert the changes to this file just keep it visible?

Reply:
I kept the removal of `ndo`, but reverted the changes that commented out the unused function parameters.